.travis.yml

```diff
@@ -24,6 +24,7 @@ script:
     if [[ $MODE == 'BreakingChange' ]]; then
       scripts/install-dotnet.sh
       npm install
+      npm install -D Azure/rest-api-specs-scripts#sergey/diff
```

Comment:
Shouldn't need this every time, since it's already in package.json

Reply:
Don't worry about that, it's an experiment. Travis-CI didn't want to install the package properly.